nilearn/datasets/utils.py

```diff
@@ -279,7 +282,7 @@ def _uncompress_file(file_, delete_archive=True, verbose=1):
     This handles zip, tar, gzip and bzip files only.
     """
     if verbose > 0:
-        print('Extracting data from %s...' % file_)
+        sys.stderr.write('Extracting data from %s...' % file_)
```

Comment:
Again... seems unnecessary to print start & end on separate lines. Follow the convention to print updating lines to stderr.